src/intersection-observer-polyfill.js

```diff
@@ -94,8 +102,12 @@ export class IntersectionObserverApi {
           this.startSendingIntersection_();
         });
 
-    this.intersectionObserver_ = new IntersectionObserverPolyfill(change => {
-      this.subscriptionApi_.send('intersection', {changes: [change]});
+    this.intersectionObserver_ = new IntersectionObserverPolyfill(entries => {
+      // Remove target info from cross origin iframe.
+      entries.forEach(entry => {
+        delete entry['target'];
```

Comment:
I always see `delete` as a codsmell, especially when it's on an object the code doesn't own.